src/main/java/by/custom/utilcalculator/domain/constants/steps/ExceptM1TransportType.java

```diff
@@ -0,0 +1,5 @@
+package by.custom.utilcalculator.domain.constants.steps;
+
+public enum ExceptM1TransportType implements StepsIndicator {
```

Comment:
Please, find and change ExceptM1 to smth like BusesAndTrucks
```suggestion
public enum BusesAndTrucksTransportType implements StepsIndicator {
```

Reply:
Done.